arrow/factory.py

```diff
@@ -167,7 +167,8 @@ def get(self, *args, **kwargs):
 
             # (None) -> now, @ utc.
             if arg is None:
-                return self.type.utcnow()
+                raise ValueError
```

Comment:
I'm OK with ValueError, but wouldn't a TypeError be more indicative?

`TypeError("Cannot parse None.")` perhaps?

Reply:
After reading [this SO post](https://stackoverflow.com/questions/48343387/valueerror-and-typeerror-in-python), I agree with Kris, this should be a `TypeError`. I think Kris's message is sufficient as well.

Reply:
Great, I used the comment you provided if you don't mind. It's done.